packages/image_picker/image_picker_ios/example/ios/RunnerTests/ImageUtilTests.m

```diff
@@ -217,4 +217,14 @@ - (void)testScaledImage_WideImage_ShouldNotBeScaledAboveOriginaWidthOrHeight {
   XCTAssertEqual(newImage.size.height, 7);
 }
 
+- (void)testScaledImage_ImageIsNil {
+  UIImage *image = nil;
+  UIImage *scaledImage = [FLTImagePickerImageUtil scaledImage:image
+                                                     maxWidth:@1440
+                                                    maxHeight:@1440
+                                          isMetadataAvailable:YES];
+
+  // Assert the same bytes pointer (not just equal objects).
+  XCTAssertEqual(image, scaledImage);
+}
```

Comment:
Can you also add a test with `maxWidth`/`maxHeight` equal to 0?

Reply:
@vashworth Thank you for your reply, i had add test. i think width or height is 0 the newImage can not be show because with and height will be 0 , so i think is need back nil.